tests/base/src/test/java/org/wildfly/security/ssl/SSLAuthenticationTest.java

```diff
@@ -514,6 +515,17 @@ public static void beforeTest() throws Exception {
                         .build(issuerSelfSignedX509CertificateAndSigningKey.getSigningKey())
         );
 
+        // Creates the CRL for rove-revoked.pem
+        X509v2CRLBuilder roveRevokedCrlBuilder = new X509v2CRLBuilder(
+                convertSunStyleToBCStyle(intermediateIssuerCertificate.getSubjectDN()),
+                currentDate
+        );
+
+        X509CRLHolder roveRevokedCrlHolder = roveRevokedCrlBuilder.setNextUpdate(nextYear).build(
+                new JcaContentSignerBuilder("SHA1withRSA")
```

Comment:
I think we want to use "SHA256withRSA" now

Reply:
Good catch Diana, yes please if we can update to SHA256withRSA

Reply:
I updated it to SHA256withRSA